File: Python/modulus1.py
"""
Program Sederhana untuk
mengetahui bilangan genap ataupun
bilangan ganjil yang diinput oleh User
"""

input_modul = int(input("Input Angka : "))
hasil = input_modul % 2

#Output Bilangan Genap
if(hasil == 0):
  print(input_modul, " Merupakan bilangan Genap")
#Output Bilangan Ganjil
elif(hasil == 1):
  print(input_modul, " Merupakan bilangan Ganjil")

File: C++/Operator/Modulus1.cpp
/*
Contoh Program Modulus
Untuk mengetahui Bilangan Genap
ataupun Ganjil yang diinput oleh User
*/

#include<iostream>
using namespace std;

int main() { 
  int modul, hasil;
  
  cout << "Input Angka : ";
  cin >> modul;
  
  //Proses mencari hasil
  hasil = modul % 2;
  
  //Output untuk Bilangan Genap
  if(hasil == 0){
    cout << modul, " Merupakan Bilangan Genap" << endl;
  }
  //Output untuk Bilangan Ganjil
  else if(hasil == 1){
    cout << modul, " Merupakan Bilangan Ganjil" << endl;
  }

  return 0;
}

